types/cesium/test/cesium-global.test.ts

```diff
@@ -96,4 +96,30 @@ const options = {
     maximumScreenSpaceError: 2
 };
 
- const tilesetModel = new Cesium.Cesium3DTilesetGraphics(options);
+const tilesetModel = new Cesium.Cesium3DTilesetGraphics(options);
+
+const polygonGraphics = new Cesium.PolygonGraphics({
+    hierarchy: Cesium.Cartesian3.fromDegreesArray([
+        -108.0,
+        42.0,
+        -100.0,
+        42.0,
+        -104.0,
+        40.0,
+    ]),
+});
+
+const polygonHierarchyProperty: Cesium.Property = polygonGraphics.hierarchy;
```

Comment:
is accessing the `hierarchy` property a sufficient test here? or should we attempt to call `getValue` or another class function on it, to guard against the break described in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44143#discussion_r423071830?

Reply:
I think this is enough to ensure that `hierarchy` is a `Property`, since otherwise this fails at compile time (the type of `polygonHierarchyProperty` is explicitly `Cesium.Property`.